index.md

```diff
@@ -83,12 +79,12 @@ a small amount of parcels we will group them into 7 types:
  - cooperatives: _registered cooperative companies like Migros, la Mobilière etc._
 
  - foundations and associations: _for example the olympic foundation for
-    cultural heritage._
+cultural heritage._
 
- - PPE: _a swiss thing: single flats owned by private people – in french_
-   proriété par étage.
+ - PPE: _the single flats in a building are owned by different people – in french
+   proriété par étage_.
```

Comment:
```suggestion
_   proriété par étage.
```
French words are italic, but in italic text, the new italic is again normal text...

Reply:
ok but weird